### PR TITLE
port using-easybuild page to MarkDown

### PR DESCRIPTION
**Pull request by boegel**
This was a pretty big effort to port, so it deserves a careful review by a 2nd pair of eyes...

### CODE REVIEW

**Review by orbsmiv — CHANGES_REQUESTED** [review]

**Comment by orbsmiv** [comment]
@boegel I'll update this PR with the changes we've made to the internal linking system so that it's consistent with the other current PRs. I'll pop it on "draft" for now but please feel free to overrule if you want to get it merged and mop the links up later!

File: docs/using-easybuild.md [hunk 1]
```diff
@@ -1,663 +1,653 @@
...
-  [...]
-  == building and installing HPCG/2.1-goolf-1.4.10...
-  [...]
-  == COMPLETED: Installation ended successfully
-  [...]
+All easyconfig file names' suffixes are `.eb` and follow format:
+`<name>-<version>-<toolchain><versionsuffix>`
 
-At this point, a module ``HPCG/2.1-goolf-1.4.10`` should have been installed.
+This is a crucial design aspect, since the dependency resolution mechanism (see [robot]()) relies upon this convention.
```
[the remark this answers is not in the document]
Comment (orbsmiv) [reply docs/using-easybuild.md]: Resolved by d65acbd

**Comment by boegel** [comment]
> @boegel I'll update this PR with the changes we've made to the internal linking system so that it's consistent with the other current PRs. I'll pop it on "draft" for now but please feel free to overrule if you want to get it merged and mop the links up later!

Makes sense to me, thanks for the effort!

**Comment by orbsmiv** [comment]
@branfosj I've looked through the whole page and also checked each link. The autorefs links, (e.g. `mkdocs_autorefs.plugin: using-easybuild.md: Could not find cross-reference target '[basic_usage_help]'`) refer to headers that will be in the auto-generated _version_specific_ files. I've created an issue (c.f. #24) to log these as and when they crop-up, although I think that we should add a placeholder file to hold all of the link-names whilst the script to generated the actual content is being written.

**Review by branfosj — APPROVED** [review]

**Pull request merged**